tox.ini

```diff
@@ -1,22 +1,32 @@
 [tox]
-envlist = py27, py35, flake8
+envlist = {py27,py35}-{default,fido}, flake8
 
 [testenv]
 deps =
     -rrequirements-dev.txt
+extras =
+    fido: fido
+setenv =
+    default: PYTEST_ADDOPTS=--ignore=tests/fido_client --ignore=tests/integration/fido_client_test.py
```

Comment:
I would prefer to have separated tests because we should make sure that the library is working as expected even without the extra ``fido`` dependency.